src/content/en/2020/seo.md

```diff
@@ -6,7 +6,7 @@ title: SEO
 description: SEO chapter of the 2020 Web Almanac covering content, meta tags, indexability, linking, speed, structured data, internationalization, SPAs, AMP and security.
 authors: [aleyda, ipullrank, fellowhuman1101]
 reviewers: [clarkeclark, natedame, catalinred, aysunakarsu, ashleyish, dsottimano, dwsmart, en3r0, Gathea, rachellcostello, ibnesayeed]
-analysts: [Tiggerito, max-ostapenko, antoineeripret]
+analysts: [Tiggerito, antoineeripret]
 translators: []
 #aleyda_bio: TODO
 #ipullrank_bio: TODO
```

Comment:
@aleyda , @ipullrank , @fellowhuman1101 can you add your Bios here which will appear at the bottom of the chapter? They should use HTML and not Markdown.

See last year's chapter as examples.

Reply:
Thanks @bazzadp ! I just added mine ... and now I'm seeing that they should use HTML and not markdown (I had used markdown). I will change that now :D

Reply:
Can you also remove the initial `#` as that is currently commenting out your bio since it wasn't there previously.

This is what it looks like when you do that:

![aleyda bio](https://user-images.githubusercontent.com/10931297/100544224-1f5ca280-324c-11eb-9cb8-40015393b7a3.png)

I think it should be Twitter rather than twitter btw.

Reply:
@bazzadp thanks Barry! Just to confirm that I did this :)

Reply:
That's better. Did you see my comment about uppercasing the T in Twitter?

Reply:
@bazzadp I hadn't! Thanks for the heads up :D Just fixed it.